packages/flutter/lib/src/cupertino/button.dart

```diff
@@ -177,11 +201,17 @@ class CupertinoButton extends StatefulWidget {
   /// If [onPressed] and [onLongPress] callbacks are null, then the button will be disabled.
   final VoidCallback? onLongPress;
 
-  /// Minimum size of the button.
+  /// Minimum Width of the button.
   ///
   /// Defaults to kMinInteractiveDimensionCupertino which the iOS Human
   /// Interface Guidelines recommends as the minimum tappable area.
-  final double? minSize;
+  final double? minWidth;
+
+  /// Minimum Height of the button.
+  ///
+  /// Defaults to kMinInteractiveDimensionCupertino which the iOS Human
```

Comment:
```suggestion
  /// Defaults to [kMinInteractiveDimensionCupertino], which the iOS Human
```

Same for `minWidth`.